app/assets/javascripts/backbone/views/loan_questions_view.coffee

```diff
@@ -34,13 +35,15 @@ class MS.Views.LoanQuestionsView extends Backbone.View
       @$('#edit-modal').modal('show')
       new MS.Views.TranslationsView(el: $('[data-content-translatable="custom_field"]'))
       @$('#custom_field_parent_id').val(parent_id)
+      $('.loan-types').select2({theme: "bootstrap"})
     )
 
   editNode: (e) ->
     id = @$(e.target).closest('li').data('id')
     @$('#edit-modal .modal-content').load("/admin/loan_questions/#{id}/edit", =>
       @$('#edit-modal').modal('show')
       new MS.Views.TranslationsView(el: $('[data-content-translatable="custom_field"]'))
+      $('.loan-types').select2({theme: "bootstrap"})
```

Comment:
I checked if select2 supports default themes, and it does:

https://select2.github.io/options.html#setting-default-options

This will remove some duplication and ensure consistency.

Also remember that you don't need `{}` inside `()` in CoffeeScript, just like Ruby.